Misc/NEWS.d/next/Library/2022-04-08-14-55-55.bpo-47256.X2mFpt.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`re` module, limit the maximum capturing group to 1,073,741,823,

```

Comment:
It is only on 64-bit platforms. On 32-bit platforms it is still 4 times less.

Reply:
Added words "in 64-bit build"